docs/source/glossary.rst

```diff
@@ -0,0 +1,29 @@
+Glossary
+========
+
+  Repository
```

Comment:
Yes, this glossary is still very incomplete.

There are also no cross references or anything else. I need to figure out how ReST and sphinx work.